### PR TITLE
Camera controls

### PR DESCRIPTION
**Pull request by martijnjanssen**

### CODE REVIEW

File: elements/camera-controls.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                }
+
+                stopCamera() {
+                    this.$.ajaxStopCamera.url = "cameras/stop";
+                    this.$.ajaxStopCamera.generateRequest();
+                }
+
+                onStopCamera(e, request) {
+                    if (request.succeeded) {
+                        console.log(request.message);
```
Comment (mennoo1996) [inline elements/camera-controls.html:98]: Why the if/else if the action is the same anyway?

**Comment by mennoo1996** [comment]
Looks good!

File: elements/camera-controls.html (hunk 1 above)
Comment (martijnjanssen) [inline elements/camera-controls.html:98]: This is to indicate we need to handle errors. Currently the frontend is only a mock, implementation will come later.

Comment (bart-de-jonge) [inline elements/camera-controls.html:98]: Then add a TODO please

File: elements/camera-controls.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                    this.$.ajaxMoveCamera.body = {
+                        direction: direction,
+                        speed: speed,
+                    };
+                    this.$.ajaxMoveCamera.generateRequest();
+                }
+
+                onMoveCamera(e, request) {
+                    if (request.succeeded) {
+                        console.log(request.message);
```
Comment (bart-de-jonge) [inline elements/camera-controls.html:141]: TODO

**Comment by bart-de-jonge** [comment]
Few small comments, other than that it looks fine to me :+1:

**Comment by martijnjanssen** [comment]
I'll add the todo comments, will be removed soon.

**Comment by Markvanderuit** [comment]
Looks good!

**Pull request merged**